packages/main/src/ColorPicker.js

```diff
@@ -0,0 +1,497 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import CSSColor from "@ui5/webcomponents-base/dist/types/CSSColor.js";
+import { isIE } from "@ui5/webcomponents-base/dist/Device.js";
+import { renderFinished } from "@ui5/webcomponents-base/dist/Render.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import Float from "@ui5/webcomponents-base/dist/types/Float.js";
+import {
+	getRGBColor,
+	HSLToRGB,
+	HEXToRGB,
+	RGBToHSL,
+} from "@ui5/webcomponents-base/dist/util/ColorConversion.js";
+import ColorPickerTemplate from "./generated/templates/ColorPickerTemplate.lit.js";
+import Input from "./Input.js";
+import Slider from "./Slider.js";
+import Label from "./Label.js";
+
+// Styles
+import ColorPickerCss from "./generated/themes/ColorPicker.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-color-picker",
+	properties: /** @lends sap.ui.webcomponents.main.ColorPicker.prototype */ {
+
+		/**
+		 * Defines the currently selected color of the <code>ui5-color-picker</code>.
+		 * *Note*:  If you need to have initially set color, or change the color programmatically, use this property
+		 * *Important*: Use HEX, RGB, RGBA, HSV, or a CSS color name when modifying this property.
+		 * @type {String}
+		 * @public
+		 */
+		color: {
+			type: CSSColor,
+			defaultValue: "rgba(255, 255, 255, 1)",
+		},
+
+		/**
+		 * Defines the HEX code of the currently selected color
+		 * *Note*: If Alpha(transperancy) is set it is not included in this property. Use <code>color</code> property.
+		 * @type {String}
+		 * @private
+		 */
+		hex: {
```

Comment:
You probably can use "noAttribute: true" to the "hex" property, but one attribute in the DOM more or less is not a big del I guess, up to you